bors.toml

```diff
@@ -0,0 +1,4 @@
+status = [ "cargo-registry-cache", "cargo-format", "cargo-sort", "cargo-check", "cargo-clippy","cargo-test", "cargo-tarpaulin","test-android-libs","build-ios-libs", "test-wasm-libs","build-linux-lib","flutter-format","flutter-checks","flutter-build-example"]
```

Comment:
Do we have to least them all or we can a leaf job in the CI graph called `bors_ok` or somthing and only list that here? Can you please take a look what other projects are doing?

Reply:
Yes, we can make a big job checking the status of all of the others and then if it is ok to merge it. I think it can look more orderly that way.

Reply:
We actually already have one: `one-leaf`, it is used to simplify a bit `notify-failure`.
We can rename it `ci-ok` and use it for both `notify-failure` and here for bors.